### ISSUE TITLE
Ideas

### ISSUE THREAD

**Issue by Mmtrx** [Mmtrx/FS25_BetterContracts#2]
A collection of ideas for future enhancements:
- don't stop mowing, fertilizing, etc contracts at 98% field work completion
- don't show all active contracts on hud. Maybe just those with completion > 0%
- add the old complete baling contracts (mow, ted, bale, wrap) as a new contract type
- dynamically configure which contract types are generated (vs. filtering out unwanted types from contract list)

Feel free to add further ideas 😉

**Comment by aurelien2023** [comment]
> - dynamically configure which contract types are generated (vs. filtering out unwanted types from contract list)

Yes, and what would be even better is to filter harvest contracts by fruit types to exclude root crops or vegetables or both. The harvest season for these contracts is too long, and the game generates too many fields with these contracts, so we end up missing some classic harvest contracts.

**Comment by Mmtrx** [comment]
Thanks for the icon! Will use from now on.
>     1. Whats "BC stand for?

BetterContracts 😄 
>     2. Need to add more money for contracts. The current rate is pathetic

will be in one of next releases, you could also try [ContractBoost](https://github.com/GMNGjoy/FS25_ContractBoost/releases)
>     3. Full grass contracts would be excellent.

will be in one of next releases
>     4. Deff need lazy NPC back ASAP

will be in one of next releases

**Comment by FirenzeIT** [comment]
Hi, it's need all features with options menu, which the same as FS 22...

**Comment by Mmtrx** [comment]
Moved ideas from here to #14. Close this to only have one stop for ideas / enhancements.
